src/main/java/ch/njol/skript/expressions/ExprLastSpawnedEntity.java

```diff
@@ -140,6 +151,9 @@ public String toString(@Nullable Event e, boolean debug) {
 			case 3:
 				word = "struck";
 				break;
+			case 4:
+				word = "launched";
+				break;
 			default:
 				throw new IllegalStateException();
```

Comment:
Rather than throwing an IllegalStateException, we can just `assert false` if this is really required. Runtime errors aren't good if we added another type to this class and this step is forgotten.

Reply:
In case it's forgotten it would be easier to figure this out by throwing a RTE than nothing (barely no one enables assertions from users)

Reply:
It's not for users, it's for our testing system to catch it when debugging the syntaxes in the chat.

Reply:
All the assert stuff Njol had was because there was a JUnit system.

Reply:
👍